packages/web-components/src/spinner/spinner.options.ts

```diff
@@ -0,0 +1,35 @@
+export type Values<T> = T[keyof T];
```

Comment:
@procload we should have access to the ValuesOf helper which was in the last published version of Foundation. I think the other option files are using it. Can we remove this and use that?

Reply:
Got it! Thanks.